src/app/components/store/storeModal.html

```diff
@@ -23,37 +27,32 @@ <h3>This is what you get</h3>
             </ul>
           </div>
         </div>
-      </div>
 
       <!-- Begin MailChimp Signup Form -->
-      <form action="https://smartcitizen.us2.list-manage.com/subscribe/post?u=d67ba8deb34a23a222ec4eb8a&amp;id=d0fd9c9327" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
-        <div class="modal_content_container" layout="row" layout-wrap>
-          <p flex="100">
+
+          <p>
             We are currently out of stock working against time to finish the new version of the Smart Citizen Kit. Share us your email below and we will notify you as soon as kits are ready!
           </p>
-          <md-input-container flex="70">
+          <md-input-container>
             <label>Email</label>
             <input type="email" name="EMAIL" required>
           </md-input-container>
 
           <div id="mce-responses" class="clear">
             <div class="response" id="mce-error-response" style="display:none"></div>
             <div class="response" id="mce-success-response" style="display:none"></div>
-          </div>    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+          </div>
         </div>
-        <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_d67ba8deb34a23a222ec4eb8a_d5a8cea29f" tabindex="-1" value=""></div>
         <md-button
            ng-disabled="MailchimpSubscriptionForm.$invalid"
            type="submit"
            name="subscribe"
            id="mc-embedded-subscribe"
-           class="md-flat md-primary main"
+           class="btn-blue btn-full"
            analytics-on="store"
            analytics-event="Kit Purchase" ng-click="addSubscription(mailchimp)" analytics-action="purchase">
-          NOTIFY ME WHEN READY
+          Notify me when ready
```

Comment:
Was this asked for?

Reply:
Capitalize is by default for the button in materialize, so that does not change on the view. And capitalize should be handled by the CSS not be the content (If you want to change later on is one line of CSS instead of hundred of HTML or worst if it's data from the database...)